File: Messenger_Client/app/src/main/java/com/messenger/main/entity/Message.kt
package com.messenger.main.entity

import com.google.gson.annotations.SerializedName

data class Message (
    @SerializedName("from_user")
    var fromUser: String,

    @SerializedName("to_user")
    var toUser: String,

    @SerializedName("msg")
    var msg: String,

    @SerializedName("date")
    var date: String
)

File: Messenger_Server/models/Message.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const MessageSchema = new Schema({
    from_user: {
        type: String,
        required: true
    },
    to_user: {
        type: String,
        required: true
    },
    msg: {
        type: String
    },
    date: {
        type: Date,
        default: Date.now
    }
});

module.exports = Message = mongoose.model('messages', MessageSchema);